code/hsec-sync/src/Security/Advisories/Sync/Atom.hs

```diff
@@ -18,7 +19,7 @@ import qualified Text.RSS.Syntax as FeedRSS
 
 latestUpdate :: String -> String -> IO (Either String UTCTime)
 latestUpdate repoUrl branch = do
-  resultE <- try $ get $ repoUrl </> "commits" </> branch </> "advisories.atom"
+  resultE <- try $ get $ mkUrl [repoUrl, "commits", branch, "advisories.atom"]
```

Comment:
ah, this is an `URL`, sure; I think with these error prone things, it would actually be useful to pull a new library, in contrast to `extra` or `either` which provide trivial combinators; are you confident `</>` is correct?

Reply:
I did not find a proper library, do you have any suggestions?

Reply:
@hasufell: I believe you had to deal with URLs/URIs in the recent past. What would you recommend?